lib/l10n/locales.dart

```diff
@@ -16,7 +16,6 @@ Iterable<LocalizationsDelegate<dynamic>> localizationsDelegates() {
 Iterable<Locale> supportedLocales() {
   return [
     const Locale('en', ''),
-    const Locale('de', ''),
```

Comment:
I found a bug here, different from the anytime player, the breez still doesn't support 'de' locale.